<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devphysiodaq_dialog</class>
 <widget class="QDialog" name="devphysiodaq_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>376</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>devphysiodaq_dialog</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>311</width>
     <height>91</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="offsetEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>50</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>enter the offset of the channel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="gainEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>enter the gain of the channel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="offsetLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>offset</string>
    </property>
   </widget>
   <widget class="QLabel" name="gainLabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>gain</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="channelLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>33</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>channel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="channelComboBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>select a channel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>200</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>leave dialog with changed settings</string>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>81</width>
     <height>32</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>leaving dialog without changed settings</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>channelComboBox</tabstop>
  <tabstop>gainEdit</tabstop>
  <tabstop>offsetEdit</tabstop>
  <tabstop>okButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
